### ISSUE TITLE
Comma in HTML dropping to new line - removes expected space in render.

### ISSUE THREAD

**Issue by dnichk** [Glavin001/atom-beautify#838]
### Description

The results of beautification are not what I expect.

Note - I may just be missing a setting!

### Expected Results

The beautified code should have looked like:

```
export const Welcome = ({name}) => (
    <div>
       Hello, {name}.
    </div>
);

```

### Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

### Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/dnichk/d4886dbcac95ebed00eb

**Comment by prettydiff** [comment]
I can reproduce this in Pretty Diff.  The new line has nothing to do with the comma and everything to do with the curly braces.  Right now it is treating the `{name}` as a markup tag relative to the neighboring markup/content and beautifying its internals as JavaScript.

I am on the fence on this one.  Normally you would want your JSX encapsulated JavaScript logic to be easier to identify from its neighboring content so long as there is not artificially injected white space.

**Comment by prettydiff** [comment]
Is this producing a defect to the HTML that is visually rendered to the user in the browser?

**Comment by dnichk** [comment]
Yes it is.

Instead of rendering “Hello, MyName”, it renders “Hello,MyName”…

**Comment by prettydiff** [comment]
Thanks,

I am looking into this now.

**Comment by prettydiff** [comment]
This is a Pretty Diff defect.  Resolution published as v1.16.26.
